modules/codegen/test/src/smithy4s/codegen/internals/TestUtils.scala

```diff
@@ -22,22 +22,53 @@ import software.amazon.smithy.model.Model
 
 object TestUtils {
 
-  def runTest(
-      smithySpec: String,
-      expectedScalaCode: String
-  )(implicit
-      loc: Location
-  ): Unit = {
+  def generateScalaCode(smithySpec: String): List[String] = {
     val model = Model
       .assembler()
       .discoverModels()
       .addUnparsedModel("foo.smithy", smithySpec)
       .assemble()
       .unwrap()
+    CodegenImpl.generate(model, None, None).map(_._2.content)
+  }
 
-    val results = CodegenImpl.generate(model, None, None)
-    val scalaResults = results.map(_._2.content)
+  def runTest(
+      smithySpec: String,
+      expectedScalaCode: String
+  )(implicit
+      loc: Location
+  ): Unit = {
+    val scalaResults = generateScalaCode(smithySpec)
     Assertions.assertEquals(scalaResults, List(expectedScalaCode))
   }
 
+  /**
+    * Finds a section starting with a given string and extracts a code section from it (based on indents)
+    * Removes empty lines too.
+    * It then asserts on expected output
+    */
+  def assertContainsSection(fileContent: String, startsWith: String)(
+      expectedSection: String
+  )(implicit loc: Location) = {
+    val lines = fileContent.lines.filter(_.trim.nonEmpty).toList.zipWithIndex
```

Comment:
CI complains about the fact that `.lines` is deprecated in 2.13 and that `.linesIterator` should be used